src/pyairios/models/vmn_05lm02.py

```diff
@@ -16,19 +19,46 @@
     U16Register,
 )
 
+LOGGER = logging.getLogger(__name__)
+
 
 class Reg(RegisterAddress):
     """Register set for VMN-05LM02 remote node."""
 
     REQUESTED_VENTILATION_SPEED = 41000
 
 
-class VMN05LM02(AiriosDevice):
-    """Represents a VMN-05LM02 remote node."""
+class NodeData(AiriosDeviceData):
```

Comment:
```suggestion
class VMN05LM02Data(AiriosDeviceData):
```

The name NodeData kind of break the hierarchy:

Node --> Device --> VMN05LM02

Reply:
Changed it to `DevData` to align. Haven't built the abstract_factory at this moment.
This class is really just data for the Module called VMN_05LM02. Having to write (almost) the same string twice invites for hard to spot errors once we get 10+ models, was my reasoning.